src/scripts/encryptLegacyPasswrd.ts

```diff
@@ -0,0 +1,18 @@
+import { setAppletPassword, PdfKey, db } from '../db'
+
+async function run() {
+  db.all(`SELECT * from pdf_keys`, [], (err, rows: PdfKey[]) => {
```

Comment:
Could we update this query to only select keys that haven't been encrypted yet, and exit early if the env var isn't set
```suggestion
  if (!process.env.AWS_KMS_KEY_ID) {
    console.warn('AWS_KMS_KEY_ID is not set, skipping encryption')
    return
  }

  db.all(`SELECT * FROM pdf_keys WHERE key NOT LIKE 'ENC_%'`, [], (err, rows: PdfKey[]) => {
```